test/CMakeLists.txt

```
@@ -5,7 +5,8 @@ llvm_canonicalize_cmake_booleans(SPIRV_SKIP_DEBUG_INFO_TESTS)
 get_target_property(LLVM_SPIRV_DIR llvm-spirv BINARY_DIR)
 set(LLVM_SPIRV_TEST_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
-if(SPIRV_TOOLS_FOUND)
+if (SPIRV_TOOLS_FOUND AND NOT SPIRV-Tools-tools_FOUND)
```

Okay, I probably have figured that out - they are the same, but SPIRV-Tools-tools is available only for `find_package` and you use it to distinguish how tools package was found, correct? If yes, may be we shouldn't call `find_package(SPIRV-Tools)` leaving only `find_package(SPIRV-Tools-tools)` ?

---

Please ignore the comment above.

@dimula73 Yet what about:
```suggestion
if (SPIRV_TOOLS_FOUND)
```
?
Because if SPIRV_TOOLS_FOUND = TRUE, then we never try to execute `find_package(SPIRV-Tools-tools)`

---

'find_package(SPIRV-Tools)' is used to add targets for shared libraries. Basically, they provide us `SPIRV_TOOLS_LDFLAGS` that is used here: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/main/tools/llvm-spirv/CMakeLists.txt#L32

And this particular line is used to distinguish between the pkgconfig and cmake paths.